### DISCUSSION TITLE
R9M does not support ELRS at 150HZ ?

### DISCUSSION THREAD

**Discussion by Gabyguay69FPV** [ExpressLRS/ExpressLRS#1817] [Q&A]
Hi everybody!150HZ is not active or has not been implemented for FRSKY R9 modules.

**Comment by whinds** [comment]
It can also do 200 Hz in theory as well, no?

**Comment by Gabyguay69FPV** [comment]
Yep, but if you are using HD VTX, can notice the differents timming on response. For that, Im using 100HZ
